application/jspmyadmin/src/main/java/com/jspmyadmin/framework/web/utils/DefaultServlet.java

```diff
@@ -114,15 +114,19 @@ public void init() throws ServletException {
 			setContext(config.getServletContext());
 			if (_context != null) {
 				setWebInfPath(_context.getRealPath("/WEB-INF/"));
-				setRootPath(_web_inf_path + "/uploads");
+				setRootPath(_web_inf_path + File.separator + "uploads");
 				File file = new File(_root_path);
-				file.setExecutable(true, false);
-				file.setReadable(true, false);
-				file.setWritable(true, false);
-				if (file.mkdirs()) {
-					_LOGGER.log(Level.INFO, "Temporary path created. Path:" + _root_path);
+				if (!file.exists()) {
+					file.setExecutable(true, false);
+					file.setReadable(true, false);
+					file.setWritable(true, false);
+					if (file.mkdirs()) {
+						_LOGGER.log(Level.INFO, "Temporary path created. Path:" + _root_path);
+					} else {
+						_LOGGER.log(Level.WARNING, "Unable to create temporary path. Path:" + _root_path);
+					}
 				} else {
-					_LOGGER.log(Level.WARNING, "Unable to create temporary path. Path:" + _root_path);
+					_LOGGER.log(Level.INFO, "Temporary path already exists. Path:" + _root_path);
```

Comment:
@Hiroyuki-Nagata : Any special reason to change this one from `WARNING` to `INFO`.

Reply:
@shashankitmaster I think it's not change, but adding process. 

Previously those lines always create a directory regardless of directory existence. According to [stackoverflow](https://stackoverflow.com/questions/27169703/mkdirs-return-value-for-already-existing-directories) , `File#mkdirs` returns false if directory was created.  Then I added this logging, because I just think it's a normal process.

Reply:
@Hiroyuki-Nagata : Got your point.